src/WebJobs.Extensions.DurableTask/HttpManagementPayload.cs

```diff
@@ -54,5 +54,23 @@ public class HttpManagementPayload
         /// </value>
         [JsonProperty("rewindPostUri")]
         public string RewindPostUri { get; internal set; }
+
+        /// <summary>
+        /// Gets the HTTP DELETE purge instance history by instance ID endpoint.
+        /// </summary>
+        /// <value>
+        /// The HTTP URL for purging instance history by instance ID.
+        /// </value>
+        [JsonProperty("purgeInstanceHitoryByInstanceIdDeleteUri")]
+        public string PurgeInstanceHitoryByInstanceIdDeleteUri { get; internal set; }
```

Comment:
Let's simplify the name to `purgeHistoryDeleteUri`.

Reply:
Thank you! I will fix it.